### ISSUE TITLE
crash on configuration screen

### ISSUE THREAD

**Issue by jdratlif** [LexManos/YUNoMakeGoodMap#27]
If you start a new game, customize the world, choose Void World, and customize that, it brings up a blank list. If you click Done, it will crash. Clicking Cancel goes back to the other screen.

Should the list be blank? I'm guessing this is why it crashed. I've attached crash logs at http://games.technoplaza.net/temp/yunomakegoodmap-crash.zip.

Minecraft 1.10.2
Forge 2099
Y U No Make Good Map 6.1.0.24

No other mods were used in this test.

Thanks.

**Comment by LexManos** [comment]
Interesting this should never bring up a blank list.
Where did you get the mod from?

**Comment by jdratlif** [comment]
I downloaded it from CurseForge website and installed into a MultiMC profile.

Just to be thorough, I also tried your adfocus official download link and getting it from the Curse Client. I even tried with the latest forge rather than 2099. On every profile, I get a blank list.

Exactly what I did to recreate:

Launch Curse
Setup new Profile 1.10.2 with Forge 2099
Install YUNoMakeGoodMap
Start Minecraft
New Singleplayer World
Customize World
Void World Customize Button
See Blank List
